test/take.js

```diff
@@ -39,4 +39,17 @@ describe('take', function() {
     eq(R.into([], R.take(0), [1, 2, 3]), []);
   });
 
+  it('steps correct number of times', function() {
+    var counter = 0;
+    var incCounter = function(x) { counter += 1; return x; };
+    var takeXTimes = function(x) {
+      counter = 0;
+      R.into([], R.compose(R.map(incCounter), R.take(x)), [1, 2, 3]);
```

Comment:
I think it would be clearer to replace `incCounter` with its definition, so `counter` could be local to `takeXTimes`.